src/modules/roc_core/target_gcc/roc_core/attributes.h

```diff
@@ -28,4 +28,18 @@
 #define ROC_ATTR_PRINTF(n_fmt_arg, n_var_arg)                                            \
     __attribute__((format(printf, n_fmt_arg, n_var_arg)))
 
+//! Used to suppress undefined behavior sanitizer for a particular function
```

Comment:
IIRC doxygen comment should be right before #define line.

Also, IIRC doxygen doesn't define macros like `__clang__` or `__GNUC__`, so the comment should be in `#else` block.

Also, doxygen comment should end with a dot (`.`), because we use auto-brief mode. In this mode, doxygen interprets the first sentence, i.e. everything before first dot, as a `@brief`.